mixer/adapter/redisquota/config/BUILD

```diff
@@ -0,0 +1,33 @@
+load("@org_pubref_rules_protobuf//gogo:rules.bzl", "gogoslick_proto_library")
```

Comment:
Please remove this file, no more Bazel in Istio.

Reply:
Removed